openmetadata-service/src/main/java/org/openmetadata/service/search/SearchClient.java

```diff
@@ -77,6 +78,8 @@ public interface SearchClient {
 
   String ADD_UPDATE_LINEAGE =
       "boolean docIdExists = false; for (int i = 0; i < ctx._source.lineage.size(); i++) { if (ctx._source.lineage[i].doc_id.equalsIgnoreCase(params.lineageData.doc_id)) { ctx._source.lineage[i] = params.lineageData; docIdExists = true; break;}}if (!docIdExists) {ctx._source.lineage.add(params.lineageData);}";
+  String ADD_UPDATE_ENTITY_RELATIONSHIP =
```

Comment:
hope we are testing this, not sure we'll be able to easily debug

Reply:
I'll check if we can add a test for it

Reply:
@OnkarVO7 whats the goal of this script?, add a comment

Reply:
@harshach this script updates the elasticsearch index. I'll add the details in the comment on code